UI/aoc-app/src/Components/SignUpForm.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { Button, Col, Form, FormGroup, Label, Input, FormText } from 'reactstrap';
```

Comment:
Remove FormText if it isn't being used. Similar to what you commented on my PR :)

Reply:
This is actually from your commit! I had to cherry-pick your commit on top and see that this

Reply:
Ya I know, I took your feedback before I merged, so you'll have a merge conflict if you don't remove it.

Reply:
I'm actually planning on dropping that commit and then rebasing on main, so I don't think that it should be an issue.